docs/reference/api/docker_remote_api_v1.25.md

```diff
@@ -406,9 +406,14 @@ Create a container
              an _absolute_ path.
            + `volume-name:container-dest` to bind-mount a volume managed by a
```

Comment:
You 're right but I 'm waiting for https://github.com/docker/docker/pull/27089 to avoid doing the same thing twice.